docs/source/release/v3.11.0/framework.rst

```diff
@@ -102,6 +102,7 @@ Python Fit Functions
 ####################
 
 - A bug that makes it difficult to define and use attributes in python fit functions has been fixed.
+- The usability of the fit functions has been improved, enabling users to construct and modify the functions as objects rather than strings.
```

Comment:
I think we should link to the new documentation here or users won't know how to get more detail on this.

Reply:
How does one do this?
The new documentation is in the concepts folder.

Reply:
You can create a link to the concept page like this: 
```
:ref:`Some text <FitFunctionsInPython>`
```

Reply:
Thank you @mantid-roman . It works.